Add estimate tx dependencies to docs (#756)
FuelLabs/fuels-rs: docs/src/calling-contracts/variable-outputs.md

> # Output variables

Reviewer: Shouldn't it be based on `VariableOutput`, i.e., keep it as `Variable outputs` (considering the client to be the source of truth here https://github.com/FuelLabs/fuel-core/blob/a3f584f12bb3e160aa00e08616a27c81734ebd02/crates/fuel-core/src/schema/tx/output.rs#L84)?

Author: Yup, I got confused

> In some cases, the contract you are calling might be transferring funds to a specific address, depending on its execution. The underlying transaction for such a contract call has to have the appropriate number of [variable outputs](https://github.com/FuelLabs/fuel-specs/blob/master/src/protocol/tx_format/output.md#outputvariable) to succeed.

Reviewer: Some conciseness nits:

```suggestion
Sometimes, the contract you call might transfer funds to a specific address, depending on its execution. The underlying transaction for such a contract call has to have the appropriate number of [variable outputs](https://github.com/FuelLabs/fuel-specs/blob/master/src/protocol/tx_format/output.md#outputvariable) to succeed.
```

Let's say you deployed a contract with the following method:

```rust,ignore
{{#include ../../../packages/fuels/tests/contracts/token_ops/src/main.sw:variable_outputs}}
```

> When calling `transfer_coins_to_output` with the SDK, you can specify the number of output variables by chaining `append_variable_outputs(amount)` to your call. Like this:

Reviewer: ```suggestion
When calling `transfer_coins_to_output` with the SDK, you can specify the number of variable outputs by chaining `append_variable_outputs(amount)` to your call. Like this:
```
